workflow/controller/operator.go

```diff
@@ -2161,6 +2161,20 @@ func (woc *wfOperationCtx) buildLocalScope(scope *wfScope, prefix string, node *
 		node = getChildNodeIndex(node, woc.wf.Status.Nodes, -1)
 	}
 
+	if node.ID != "" {
+		key := fmt.Sprintf("%s.id", prefix)
+		scope.addParamToScope(key, node.ID)
+	}
+
+	if !node.StartedAt.Time.IsZero() {
+		key := fmt.Sprintf("%s.startedAt", prefix)
+		scope.addParamToScope(key, node.StartedAt.Time.Format(time.RFC3339))
+	}
```

Comment:
Minor: space after clause
```suggestion
	}

```

Reply:
fixed to insert new line